chia/wallet/wallet.py

```diff
@@ -307,8 +307,8 @@ async def _generate_unsigned_transaction(
         coins: Set[Coin] = None,
         primaries_input: Optional[List[AmountWithPuzzlehash]] = None,
         ignore_max_send_amount: bool = False,
-        coin_announcements_to_consume: Set[Announcement] = None,
-        puzzle_announcements_to_consume: Set[Announcement] = None,
+        coin_announcements_to_consume_as_bytes: Optional[Set[bytes32]] = None,
+        puzzle_announcements_to_consume_as_bytes: Optional[Set[bytes32]] = None,
```

Comment:
Pretty sure this is a bad merge, we switched to having these arguments be Announcements when we merged in the wallet changes.  There's lots of other code that depends on this.

Reply:
And all of the other changes in this file too.

Reply:
I see - so all calls into Wallet.generate_signed_transaction have been changed to use Announcements?

Reply:
Correct